msteams/msteams.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+
+import json
+import http.client
+import urllib.parse
+import sys
+
+
+# Function to send an HTTP POST request
+def send_http_request(webhook_url, headers, payload):
+    parsed_url = urllib.parse.urlparse(webhook_url)
+    host = parsed_url.netloc
+    url_path = parsed_url.path
+
+    conn = http.client.HTTPSConnection(host)
+    conn.request("POST", url_path, body=payload, headers=headers)
+    res = conn.getresponse()
+    data = res.read()
+    conn.close()
+
+    return res.status, data.decode('utf-8')
+
+
+# Function to send message to Microsoft Teams
+def send_teams_message(webhook_url, message):
+    payload = json.dumps({
+        '@type': 'MessageCard',
+        '@context': 'http://schema.org/extensions',
+        'summary': 'RPort Notification',
+        'sections': [{
+            'activityTitle': 'RPort Notification',
+            'text': message
+        }]
+    })
+
+    headers = {'Content-Type': 'application/json'}
+    status, response_data = send_http_request(webhook_url, headers, payload)
+
+    if status == 200:
+        print(f"Successfully sent message to Teams: {message}")
+    else:
+        sys.stderr.write(f"Failed to send message to Teams. Status code: {status}, Reason: {response_data}\n")
+        exit(1)
+
+
+if __name__ == "__main__":
+    # Read JSON from stdin
+    input_json = json.load(sys.stdin)
```

Comment:
@KonradKuznicki Shouldn't we catch and handle json decoding errors?
If people try the script on the console, we can't rely on the input always being proper json.

Reply:
python is going to display stack trace with error message if JSON is invalid that is fairly decent,
On the other hand I think that examples should be quite simple so the user can clearly see the most important parts